test/com/google/enterprise/adaptor/filenet/PermissionsTest.java

```diff
@@ -691,4 +694,275 @@ public void testMarking_UserNotMatchingAnyAces() {
         SecurityPrincipalType.USER, "user2@bar.example.com", user1, false,
         AccessRight.NONE);
   }
+
+  //======================================================================
```

Comment:
TODO(bmj): Delete this delimiting comment when removing the authorizeMarking tests above.